test/e2e/node/pods.go

```diff
@@ -338,7 +338,7 @@ var _ = SIGDescribe("Pods Extended", func() {
 				return podClient.Delete(ctx, pod.Name, metav1.DeleteOptions{})
 			})
 
-			err := e2epod.WaitForPodTerminatedInNamespace(ctx, f.ClientSet, pod.Name, "Evicted", f.Namespace.Name)
+			err := e2epod.WaitForPodTerminatedInNamespaceTimeout(ctx, f.ClientSet, pod.Name, "Evicted", f.Namespace.Name, 10*time.Minute)
```

Comment:
why only these tests are affected? Is it because these tests rely on metrics availability? If so, can you please add a comment in code?

Reply:
Yes, that's correct. I added a comment to clarify that.